packages/jest-circus/src/__tests__/__snapshots__/hooks.test.js.snap

```diff
@@ -0,0 +1,64 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`beforeEach is executed before each test in current/child describe blocks 1`] = `
+"start_describe_definition: describe
+add_hook: beforeEach
+add_test: one
+add_test: two
+start_describe_definition: 2nd level describe
```

Comment:
This test would be even better if it had the depth represented with indents. Would you mind adding that?